src/model/base.go

```diff
@@ -1,9 +1,7 @@
 package model
 
 import (
-	"fmt"
```

Comment:
Subor `model.go` som rozdelil na `keys.go`, `base.go`, `rename.go` ... kedze tam bolo toho uz vela a bolo to neprehladne.